### PR TITLE
Add DataCollectionFragment skeleton

### PR DESCRIPTION
**Pull request by JSunde**
Adding a new DataCollection screen and a temporary button in the navbar to launch it.
Towards ##1146

@gino-m PTAL?

### CODE REVIEW

File: gnd/src/main/java/com/google/android/gnd/ui/datacollection/DataCollectionFragment.kt [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2021 Google LLC
```
Comment (gino-m) [inline gnd/src/main/java/com/google/android/gnd/ui/datacollection/DataCollectionFragment.kt:2]: Nit: -> 2022

Comment (JSunde) [reply gnd/src/main/java/com/google/android/gnd/ui/datacollection/DataCollectionFragment.kt]: Done!

File: gnd/src/main/res/menu/nav_drawer_menu.xml [hunk 2]
```diff
@@ -35,6 +35,10 @@
   <group
     android:id="@+id/group_other"
     android:checkableBehavior="none">
+    <item
```
Comment (gino-m) [inline gnd/src/main/res/menu/nav_drawer_menu.xml:38]: Please add a comment referencing a GitHub issue (`TODO(#...):`) to remove this menu item once the actual flow is implemented.

File: gnd/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -138,6 +138,7 @@
   <string name="sign_out">Sign out</string>
   <string name="cancel">Cancel</string>
   <string name="join_survey">Join survey</string>
+  <string name="collect_data">Collect data</string>
```
Comment (gino-m) [inline gnd/src/main/res/values/strings.xml:141]: Consider renaming to something that makes it more obvious this is a temporary hook (eg "Test new collect data UI")

**Review by gino-m — CHANGES_REQUESTED** [review]

File: gnd/src/main/res/menu/nav_drawer_menu.xml (hunk 2 above)
Comment (JSunde) [reply gnd/src/main/res/menu/nav_drawer_menu.xml:39]: Done!

File: gnd/src/main/res/values/strings.xml (hunk 3 above)
Comment (gino-m) [reply gnd/src/main/res/values/strings.xml:143]: Sgtm.

**Review by gino-m — APPROVED** [review]

**Pull request merged**